src/components/CardsContainer.tsx

```diff
@@ -1,28 +1,78 @@
 'use client'
 
 import UserCard from './UserCard'
+import { useState, useEffect } from 'react'
+
 import { namesArray } from '@/utils/sampleData'
 import { sealifeImages } from '@/utils/imageLists'
+// import ChatQuestionOutline from './icons/JellyFishOutlineIcon'
+import { useAllUsersPointsListener } from '@/services/socket'
+import { useSocketListener } from '@/services/socket'
+import type { ListenerRes } from '@/services/socket'
 
 export default function CardsContainer() {
-	const sampleNames = namesArray.slice(10, 30)
+	const [usersPointsData, setUsersPointsData] = useState<ListenerRes[]>([])
+	const [showStoryPoints, setShowStoryPoints] = useState(false)
+	// const sampleNames = namesArray.slice(0, 9)
+	const sampleNames: string[] = []
 	const storyPointsArray = [0, 1, 2, 3, 5, 8, 13]
+	const allUsersPoints = useAllUsersPointsListener('all-users-story-points')
```

Comment:
todo/bug: This pattern here of using a hook to get some data, and then use useEffect to create a new state... Doesn't this sound complex to you? We can pair about it in our call.

Reply:
I'll work on this one next and do another commit

Reply:
I'll send a video. Trying to use the pattern we discussed causes an infinite loop. At least I think it does.